errors.go

```diff
@@ -59,6 +59,7 @@ var (
 	ErrStartTimeLaterThanEndTime     = errors.New("gocron: WithStartDateTime: start must not be later than end")
 	ErrStopTimeEarlierThanStartTime  = errors.New("gocron: WithStopDateTime: end must not be earlier than start")
 	ErrWithStopTimeoutZeroOrNegative = errors.New("gocron: WithStopTimeout: timeout must be greater than 0")
+	ErrSchedulerMonitorNil           = errors.New("gocron: scheduler monitor cannot be nil")
```

Comment:
```suggestion
	ErrWithSchedulerMonitorNil           = errors.New("gocron: WithSchedulerMonitor: scheduler monitor cannot be nil")
```
Also, please keep this list of errors alphabetical